src/debug_menu.cpp

```diff
@@ -1791,21 +1791,36 @@ static void teleport_overmap( bool specific_coordinates = false )
                    coord_strings.size() );
             return;
         }
-        std::vector<int> coord_ints;
+        std::vector<std::pair<int, int>> coord_ints
```

Comment:
```suggestion
        std::vector<std::pair<int, int>> coord_ints;
```
Apologies but some tests had frozen so I did this to try and unfreeze them.